cibuildwheel/util.py

```diff
@@ -430,6 +430,11 @@ class FileReport(NamedTuple):
         FileReport(wheel.name, f"{(wheel.stat().st_size + 1023) // 1024:,d}")
         for wheel in final_contents - existing_contents
     ]
+
+    if len(new_contents) == 0:
+        print("No new wheels")
```

Comment:
```suggestion
```

Let's remove this message as there's already a message printed when no build identifiers are selected.